Disambiguate 'Linux' from 'Ubuntu' (#1279)
ros2/ros2_documentation: source/Guides/Cross-compilation.rst

Cross-compiling examples for Arm
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> After `downloading the ROS 2 source code <../../Installation/Ubuntu-Development-Setup>`__, you can add cross-compilation assets to the workspace via ``git clone https://github.com/ros-tooling/cross_compile.git -b 0.0.1 src/ros2/cross_compile``. These are working examples on how to cross-compile for Arm cores.

Reviewer: Since this is a local link, it shouldn't have the double-underscore.

```suggestion
After `downloading the ROS 2 source code <../Installation/Ubuntu-Development-Setup>`, you can add cross-compilation assets to the workspace via ``git clone https://github.com/ros-tooling/cross_compile.git -b 0.0.1 src/ros2/cross_compile``. These are working examples on how to cross-compile for Arm cores.
```

    mkdir qemu-user-static
    cp /usr/bin/qemu-*-static qemu-user-static

> The standard `setup <../../Installation/Ubuntu-Development-Setup>`__ process of ROS 2 is run inside an arm docker. This is possible thanks to ``qemu-static``, which will emulate an arm machine. The base image used is an Ubuntu Bionic from Docker Hub.

Reviewer: Same.

```suggestion
The standard `setup <../Installation/Ubuntu-Development-Setup>` process of ROS 2 is run inside an arm docker. This is possible thanks to ``qemu-static``, which will emulate an arm machine. The base image used is an Ubuntu Bionic from Docker Hub.
```

    git clone https://github.com/ros-tooling/cross_compile.git -b 0.0.1
    cd ..

> Generate and export the file-system as described in `3. Prepare the sysroot`_, but with the provided ``Dockerfile_ubuntu_arm64_prebuilt``. These ``_prebuilt`` Dockerfile will use the `binary packages <https://index.ros.org/doc/ros2/Ubuntu-Install-Debians/>`__ to install ROS 2 instead of building from source.

Reviewer: ```suggestion
Generate and export the file-system as described in `3. Prepare the sysroot`_, but with the provided ``Dockerfile_ubuntu_arm64_prebuilt``. These ``_prebuilt`` Dockerfile will use the `binary packages <../Installation/Ubuntu-Install-Debians>` to install ROS 2 instead of building from source.
```
